internal/context/context.go

```diff
@@ -135,24 +135,32 @@ func makeDir(filePath string) error {
 	return nil
 }
 
-func SignNFCert(nfType, nfId string) error {
-	nfCertPath := openapi.GetNFCertPath(factory.NrfConfig.GetCertBasePath(), nfType)
+func SignNFCert(nfType, nfId, nfCertPem string) error {
```

Comment:
what is `nfCertPem` used for?

Reply:
It is not used now. Would be removed in next commit.